database/dqlite/dqlite_other.go

```diff
@@ -0,0 +1,16 @@
+//go:build !dqlite
+
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package dqlite
+
+type NodeInfo struct {
+	ID      uint64
+	Address string
+	Role    int
+}
```

Comment:
Can we add `yaml` tags to this. Once we update the dependency we should already be set.

Reply:
Given that we need new static assets, I'll do this in the next patch and land this one if I can get it to pass.